packages/cli/src/templates/shopware-pwa.config.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
   shopwareEndpoint: "<%= props.shopwareEndpoint %>",
-  shopwareAccessToken: "<%= props.shopwareAccessToken %>",
+  shopwareAccessToken: "<%= props.shopwareAccessToken %>"
```

Comment:
prettier shouldn't allow that
```suggestion
  shopwareAccessToken: "<%= props.shopwareAccessToken %>",
```